website/client/src/components/HeaderBar/AuthenBar.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import LoginModal from '../Modal/LoginModal';
+import { connect } from 'react-redux';
+import './HeaderBar.scss';
+import RegisterModal from '../Modal/RegisterModal';
+import Button from '../Button/Button';
+import { getUser } from '../../actions/userAction';
+import UserAva from './UserAva';
+
+let isOpenModalRegister = false;
```

Comment:
Why not using state to manage this variable? You're mutating the variable and call forceUpdate that will confuse React component to render the dom.